config/_default/menus.yaml

```diff
@@ -11,139 +11,150 @@
 # ordered by weight from lowest to highest.
 
 main:
+
+  # About tab
   - name: About
-    weight: 10
+    weight: 1
```

Comment:
I rather liked the spacing by 10 better - it's a lot easier to add a new item without having to reorder everything, i.e., a 15 works!